i18next-parser.config.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
-  locales: ["en", "ja"],
+  locales: ["ja"],
```

Comment:
```suggestion
  locales: ["en"],
```

I would prefer to generate `en.yml` as well.

This problem is also a question of how to name the keys. We have adopted the method of using the English text as the key. The advantage of this is that we do not have to write the English text again in the YAML file. The disadvantage is not so much that it is difficult to translate the same English text into different texts.

By generating `en.yml` and leaving the keys lined up but leaving the translations empty, you can write the English text in the source code and when the same English text is used, you can change the key names slightly and write the correct English text in the `en.yml` file. If the key is empty, the key will be displayed on the screen as it is, according to the i18next settings.

Reply:
I reverted `en.yml` at this commit :pray:
[Revert "fix: use key as english translation"](https://github.com/reearth/reearth-web/pull/240/commits/ff56843132be0b8d85e5eb974d479901e9940ebf)